### PR TITLE
Added Azure SQL DB as unsupported source server for package files.

### PR DESCRIPTION
**Pull request by GarrySinghMSFT**
Azure SQL DB cannot be utilized as source catalog for ispac files. It seems to only work with SQL Server on-premise
 - Error when using Azure SQL DB. "Failed to connect to server barcelona.database.windows.net."
 - There is no option to specify any credentials.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@GarrySinghMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/integration-services/lift-shift/ssis-azure-deploy-run-monitor-tutorial.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ To learn more about deploying packages and about the Deployment Wizard, see [Dep
 
 2. On the **Select Source** page, select the existing SSIS project to deploy.
     -   To deploy a project deployment file that you created, select **Project deployment file** and enter the path to the .ispac file.
-    -   To deploy a project that resides in an SSIS catalog, select **Integration Services catalog**, and then enter the server name and the path to the project in the catalog.
+    -   To deploy a project that resides in an SSIS catalog, select **Integration Services catalog**, and then enter the server name and the path to the project in the catalog. (This step is not unsupported with Azure SQL DB)
```
Comment (swinarko) [inline docs/integration-services/lift-shift/ssis-azure-deploy-run-monitor-tutorial.md:88]: Please REPLACE
"(This step is not unsupported with Azure SQL DB)"
WITH
"Only projects residing in SSISDB hosted by SQL Server can be redeployed in this step."

**Review by swinarko — CHANGES_REQUESTED** [review]

**Comment by PRMerger19** [comment]
@GarrySinghMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by GarrySinghMSFT — COMMENTED** [review]
Made Suggested Changes.

**Comment by swinarko** [comment]
#sign-off

**Review by swinarko — APPROVED** [review]

**Comment by swinarko** [comment]
#sign-off

**Comment by PRMerger19** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**